app/src/features/Batches/BatchList.tsx

```diff
@@ -0,0 +1,160 @@
+      margin: 0,
+      transition: "none",
+      "&.Mui-expanded": {
+        minHeight: 56,
+      },
+    },
+    "&.MuiAccordionSummary-root": {
+      borderBottom: `1px solid ${theme.palette.background.default}`,
+      minHeight: 56,
+      margin: 0,
+      transition: "none",
+    },
+  },
+  button: {
+    textTransform: "none",
+    marginLeft: theme.spacing(1),
+  },
+  accordionDetails: {
+    display: "flex",
+    flexDirection: "column",
+    marginTop: theme.spacing(0.5),
+    marginBottom: theme.spacing(1),
+  },
+  accordion: {
+    "&.Mui-expanded": {
+      margin: 0,
+      marginBottom: theme.spacing(0.5),
+    },
+    "&:before": {
+      display: "none",
+    },
+    borderRadius: theme.shape.borderRadius,
+    marginBottom: theme.spacing(0.5),
+  },
+  pagination: {
+    margin: `${theme.spacing(1)}px 0px`,
+  },
+}));
+
+const PAGE_SIZE = 10;
+
+const BatchList = (): JSX.Element => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+
+  const [page, setPage] = useState(1);
+  const handlePageChange = (
+    event: React.ChangeEvent<unknown>,
+    value: number
+  ) => {
+    setPage(value);
+  };
+
+  const offset = (page - 1) * PAGE_SIZE;
+  const limit = offset + PAGE_SIZE;
+  const { data: batches, isLoading: isBatchesLoading } = useApiBatchesListQuery(
+    {
+      limit,
+      offset,
+      ordering: "-created_at",
+    }
+  );
+
+  const handleBatchRetry = (batchId: string) => () => {
+    // TODO: batch retry is not implemented yet
+    console.log(batchId);
+  };
+
+  if (isBatchesLoading) return <CircularProgress />;
+  return (
+    <>
+      {batches?.results &&
+        batches.results.map((batch) => (
+          <Accordion key={`batch-${batch.id}`} className={classes.accordion}>
+            <AccordionSummary
+              className={classes.accordionSummary}
+              expandIcon={<ExpandMore />}
+              aria-controls="panel1a-content"
+              id="panel1a-header"
+            >
+              <div className={classes.accordionSummaryContent}>
+                <Typography variant="subtitle2">
+                  {new Date(batch.created_at).toLocaleString()}
+                </Typography>
+                <div>
+                  <Button
+                    className={classes.button}
+                    variant="contained"
+                    color="primary"
+                    startIcon={<Replay />}
+                    onClick={(e) => {
+                      e.stopPropagation();
+                      handleBatchRetry(batch.id);
+                    }}
+                  >
+                    <Typography>{t("retry")}</Typography>
+                  </Button>
+                  {!batch.deleted_at && (
+                    <BatchCancel
+                      batch={batch}
+                      className={classes.button}
+                      variant="contained"
+                      color="primary"
+                      startIcon={<CancelOutlined />}
```

Comment:
Same than for `CreateSource`, those styling props could already be defined inside `BatchCancel`